pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -1482,13 +1482,14 @@ func maybeRemoveStaleIpsets(staleIpsets []string) {
 	// domUs, then the kernel would not remove them.
 	// The ipset destroy command would just fail.
 	for _, ipset := range staleIpsets {
```

Comment:
Same ipset/hostname argument name confusion is possible here. I wonder how pervasive it would be to rename it all.

Alternatively, we can document in commentsthat the abstract ipset name is without the ipv4./ipv6. prefix (calling it a baseipsetname?).

Reply:
I have performed the renaming at least for `ipset.go` and for functions touched by this PR (like `maybeRemoveStaleIpsets`)
There are several uses of `ipsets []string` across zedrouter which actually refer to list of hostnames used for ipsets, hence the proposed `ipsetHosts []string` might be better. I feel that this PR which is meant to fix a bug would grow in size quite a bit, hence I would propose to do the renaming in a separate PR. Part of that PR would be also a refactor of [this code](https://github.com/lf-edge/eve/blob/master/pkg/pillar/cmd/zedrouter/acl.go#L106-L229), which suffers from the issue that I have described on the Slack (i.e. it doesn't build the set of ipsets correctly under certain scenarios).